api/storage_service.cc

```diff
@@ -1670,6 +1670,18 @@ rest_raft_topology_upgrade_status(sharded<service::storage_service>& ss, std::un
         co_return sstring(format("{}", ustate));
 }
 
+static
+future<json::json_return_type>
+rest_raft_topology_get_cmd_status(sharded<service::storage_service>& ss, std::unique_ptr<http::request> req) {
+        const auto status = co_await ss.invoke_on(0, [] (auto& ss) {
+            return ss.get_topology_cmd_status();
+        });
+        if (status.active_dst.empty()) {
+            co_return sstring("none");
+        }
+        co_return sstring(fmt::format("{}[{}]: {}", status.current, status.index, fmt::join(status.active_dst, ",")));
```

Comment:
I get that it's for the debugging purposes, but I think we should still make sure that the meaning of the returned string is clear. We could specify in `"summary":` that the command type, the command index, and active destinations are returned. Another option is to add some words to the string.

Reply:
I am not sure I agree. Adding non essential information here will just make it harder to parse by a script for instance.